src/linear_algebra/affine_space.lean

```diff
@@ -552,23 +542,23 @@ begin
       rw set.sum_indicator_subset _ (finset.subset_union_right s1 s2) at hw2,
       have hws : ∑ i in s1 ∪ s2, (set.indicator ↑s1 w1 - set.indicator ↑s2 w2) i = 0,
       { simp [hw1, hw2] },
-      rw [finset.affine_combination_indicator_subset _ _ _ (finset.subset_union_left s1 s2),
-          finset.affine_combination_indicator_subset _ _ _ (finset.subset_union_right s1 s2),
-          ←vsub_eq_zero_iff_eq V, finset.affine_combination_vsub] at heq,
+      rw [finset.affine_combination_indicator_subset _ _ (finset.subset_union_left s1 s2),
+          finset.affine_combination_indicator_subset _ _ (finset.subset_union_right s1 s2),
+          ←@vsub_eq_zero_iff_eq V, finset.affine_combination_vsub] at heq,
```

Comment:
Why is the `@` needed here?

Reply:
`rw` fails otherwise. I don't understand why.